ibis-server/app/config.py

```diff
@@ -58,6 +58,13 @@ def update(self, diagnose: bool):
         else:
             self.init_logger()
 
+    def get_remote_function_list_path(self, data_source: str) -> str:
+        return (
+            f"{self.remote_function_list_path}/{data_source}/functions.csv"
```

Comment:
maybe change to this pattern? We don't need another folder, I guess.
```suggestion
            f"{self.remote_function_list_path}/{data_source}.csv"
```

Reply:
Sure